op/clip/shapes.go

```diff
@@ -26,6 +26,23 @@ func (r Rect) Add(ops *op.Ops) {
 	r.Op().Add(ops)
 }
 
+// Path returns the PathSpec for the rectangle.
+func (r Rect) Path(ops *op.Ops) PathSpec {
+	var p Path
```

Comment:
This results in a generic path clip, whereas Rect.Op records a very cheap pixel-aligned clip. One solution (other than leaving out Rect.Path altogether) is to move the bounds field from Op to Path, so that this Rect.Path method can simply return

```
PathSpec{
    bounds: image.Rectangle(r),
}
```

I'm going to need PathSpec bounds for all paths for compute anyway, so moving the field is probably going to happen regardless.

Reply:
I'll remove it from this changeset for now.

I'm not yet quite sure how that change would work together with `clip.Stroke`.